ligo/src/helper/unpack.mligo

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#if !HELPER_CONFIG_H
+#define HELPER_CONFIG_H
+
+#include "../types.mligo"
+
+// ------------------------------
+// Unpack Helpers
+// ------------------------------
+
+let unpack_key (key_name, p : string * ((string, bytes) map)) : bytes =
+    match Map.find_opt key_name p with
+      Some (b) -> b
+    | None -> (failwith "UNPACK_KEY_NOT_FOUND" : bytes)
+
+let unpack_nat (b: bytes) : nat =
+  match ((Bytes.unpack b) : (nat option)) with
```

Comment:
Small improvement, it seems that all the following `unpack` helpers (including this one and `unpack_transfer_type_list`) actually need `unpack_key` to be used, so why not modify them to include it instead and remove the repetition at the calling side, e.g.:
```suggestion
let unpack_nat (key_name, p : string * ((string, bytes) map)) : nat =
  let b = unpack_key(key_name, p) in
  match ((Bytes.unpack b) : (nat option)) with
```